testdata/baselines/reference/submodule/conformance/symbolDeclarationEmit12.errors.txt.diff

```diff
@@ -1,26 +1,20 @@
 --- old.symbolDeclarationEmit12.errors.txt
 +++ new.symbolDeclarationEmit12.errors.txt
-@@= skipped -0, +-1 lines =@@
--symbolDeclarationEmit12.ts(9,13): error TS2300: Duplicate identifier '[Symbol.toPrimitive]'.
--symbolDeclarationEmit12.ts(10,13): error TS2300: Duplicate identifier '[Symbol.toPrimitive]'.
--
--
+@@= skipped -0, +0 lines =@@
++symbolDeclarationEmit12.ts(5,9): error TS2300: Duplicate identifier '[Symbol.toPrimitive]'.
```

Comment:
It's because those are reported on *all* duplicate declarations and not just on non-first ones. This is desired as per https://github.com/microsoft/typescript-go/pull/608#issuecomment-2731190882